### PR TITLE
feat: added footer

### PR DESCRIPTION
**Pull request by Anuragwagh**

### CODE REVIEW

File: data/users/AnuragWagh.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{
+    "name": "Anurag Wagh",
```
Comment (codewithvoid) [inline data/users/AnuragWagh.json:2]: revert this file: send in a separate change like [this](https://github.com/codewithvoid/AskDevs/pull/196)

File: public/categories.json [hunk 2]
```diff
@@ -30,5 +30,9 @@
   {
     "name": "Blockchain Development",
     "slug": "blockchain-development"
+  },
+  {
+    "name": "AI / ML",
+    "slug": "AI-ML"
```
Comment (codewithvoid) [inline public/categories.json:36]: you can use `data-science` slug instead of adding this

File: src/components/Footer.js [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+      <a class="flex title-font font-medium items-center md:justify-start justify-center text-gray-900">
+      <HeaderLogo />
+      </a>
+    </div>
+    <div class="flex-grow flex flex-wrap md:pl-20 -mb-10 md:mt-0 mt-10 md:text-left text-center">
+      <div class="lg:w-1/3 md:w-1/2 w-full px-12">
+      <h2 class="title-font font-large text-white tracking-widest text-xl mb-3">Connect</h2>
+        <nav class="list-none mb-10">
+          <li>
+            <a class="title-font font-medium text-white tracking-widest text-xlm mb-3" href='https://twitter.com/codewithvoid'>twitter</a>
```
Comment (codewithvoid) [inline src/components/Footer.js:21]: capitalize Twitter text here

File: yarn.lock [hunk 4]
```diff
@@ -0,0 +1,3606 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
+# yarn lockfile v1
```
Comment (codewithvoid) [inline yarn.lock:2]: can you revert this..we support package-lock.json for the project

**Review by codewithvoid — CHANGES_REQUESTED** [review]
Thanks a ton for making the changes. Left few comments.

**Comment by codewithvoid** [comment]
@Anuragwagh Let me know if you are facing any issue while making requested changes.

**Comment by Anuragwagh** [comment]
I'm not sure what exactly has to do with package json file it will great if you provide guidance

**Pull request closed without merging**